src/routes.js

```diff
@@ -1,7 +1,10 @@
+/* eslint no-undef: "off" */
```

Comment:
Required as eslint will trigger an error for the undefined property “CustomEvent” in this file. CustomEvent is only defined on the clientside.